packages/web/lib/networking/mutations/subscribeMutation.ts

```diff
@@ -11,16 +14,22 @@ type Subscribe = {
   subscriptions: Subscription[]
 }
 
+export type SubscribeMutationInput = {
+  name?: string
+  url?: string
+  subscriptionType?: SubscriptionType
+}
+
 export async function subscribeMutation(
-  subscribeName: string
+  input: SubscribeMutationInput
 ): Promise<any | undefined> {
   const mutation = gql`
-    mutation {
-      subscribe(name: "${subscribeName}") {
+    mutation Subscribe($input: SubscribeInput!) {
```

Comment:
nice, using the string method like was here before isn't recommended since things like `undefined` are not handled well.